book/src/prevent-volume-mode-conversion.md

```diff
@@ -0,0 +1,31 @@
+# Prevent unauthorised volume mode conversion
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | external-snapshotter Version | external-provisioner Version
+--|--|--|--|--
+Alpha | 1.24 | - | 3.2+ | 6.0.1+
+
```

Comment:
Add a few sentences here talking about this feature is only added to v1.  v1beta1 is no longer supported.